### PR TITLE
add Predy V5

### PR DESCRIPTION
**Pull request by wshino**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by wshino** [comment]
```
% node test.js projects/predy-v5/index.js
arbitrum summing tokens 2
--- arbitrum ---
USDC                      130.08 k
WETH                      117.56 k
Total: 247.64 k 

--- arbitrum-borrowed ---
WETH                      13.93 k
ARB                       3.75 k
LUSD                      492.0305935912003
GYEN                      25.06652624599175
Total: 18.20 k 

--- tvl ---
USDC                      130.08 k
WETH                      117.56 k
Total: 247.64 k 

--- borrowed ---
WETH                      13.93 k
ARB                       3.75 k
LUSD                      492
GYEN                      25
Total: 18.20 k 

------ TVL ------
arbitrum                  247.64 k
arbitrum-borrowed         18.20 k
borrowed                  18.20 k

total                    247.64 k 
```

File: projects/predy-v5/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    // ID = 10,11,12,13
+
+    // V5
+    const WETH = await api.call({ abi: abi.v5.getAsset, target: v5Address, params: WETH_ASSET_ID})
+    const ARB = await api.call({ abi: abi.v5.getAsset, target: v5Address, params: ARB_ASSET_ID })
+    const GYEN = await api.call({ abi: abi.v5.getAsset, target: v5Address, params: GYEN_ASSET_ID })
+    const LUSD = await api.call({ abi: abi.v5.getAsset, target: v5Address, params: LUSD_ASSET_ID })
+    const WETHEX = await api.call({ abi: abi.v5.getAsset, target: v5Address, params: WETHEX_ASSET_ID })
+    
+    const WETHBorrowed = (new BigNumber(WETH[5][4])).toNumber()
```
Comment (predy-dev) [inline projects/predy-v5/index.js]: pair毎にUSDCプールもありまして、以下のようになります。

usdcBorrowedForWETH_USDCe_Pair = WETH[2][2][2]
wethBorrowedForWETH_USDCe_Pair = WETH[3][2][2]

Comment (wshino) [reply projects/predy-v5/index.js]: @syuhei176 
ABIを更新するのを忘れていました。

Duneの値
Borrow: 46720
Supply: 257975

Defillamaの値
arbitrum                  253.23 k
arbitrum-borrowed         46.72 k


Supplyが4kくらい違います。何か見落としているものがあれば教えて欲しいです。

```
% node test.js projects/predy-v5/index.js
arbitrum summing tokens 5
--- arbitrum ---
WETH                      119.40 k
USDC                      116.04 k
ARB                       15.62 k
LUSD                      1.47 k
GYEN                      696.1572795209217
Total: 253.23 k 

--- arbitrum-borrowed ---
USDC                      31.64 k
WETH                      10.84 k
ARB                       3.72 k
LUSD                      492.6299547166729
GYEN                      25.011403440074748
Total: 46.72 k 

--- tvl ---
WETH                      119.40 k
USDC                      116.04 k
ARB                       15.62 k
LUSD                      1.47 k
GYEN                      696
Total: 253.23 k 

--- borrowed ---
USDC                      31.64 k
WETH                      10.84 k
ARB                       3.72 k
LUSD                      493
GYEN                      25
Total: 46.72 k 

------ TVL ------
arbitrum                  253.23 k
arbitrum-borrowed         46.72 k
borrowed                  46.72 k

total                    253.23 k 
```

Comment (predy-dev) [reply projects/predy-v5/index.js]: 証拠金も合わせると、数字見る限り大体合っていると思います。

**Review by predy-dev — APPROVED** [review]
LGTM

**Pull request merged**